docs/bridge_service.md

```diff
@@ -4,25 +4,115 @@ The bridge service abstracts interaction with the unified LxLy bridge. It repres
 
 ## Bridge flow
 
-The diagram below describes the basic L2 -> L2 bridge workflow. Note that L2 networks consist of the `aggkit` node and execution client.
+Note that L2 networks consist of the `aggkit` node and execution client.
```

Comment:
Now that we have a different participant for the `Aggkit`, we don't need this disclaimer, I'd say.
```suggestion
```